ts/import-log/QueueSummary.svelte

```diff
@@ -0,0 +1,40 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import * as tr from "@tslib/ftl";
+
+    import IconConstrain from "../components/IconConstrain.svelte";
+    import { showInBrowser } from "./lib";
+    import type { SummarizedLogQueues } from "./types";
+
+    export let summary: SummarizedLogQueues;
+
+    $: notes = summary.queues.map((queue) => queue.notes).flat();
+
+    function onShow(event: MouseEvent) {
+        showInBrowser(notes.map((note) => note?.id));
+        event.preventDefault();
+    }
+</script>
+
+{#if notes.length}
+    <li>
+        <IconConstrain>
+            {@html summary.icon}
+        </IconConstrain>
+        {summary.summary_template({ count: notes.length })}
+        {#if summary.canBrowse}
+            (
+            <a href="/#" tabindex="-1" on:click={onShow}>{tr.importingShow()}</a>
```

Comment:
How about updating importingShow() to be "Show", and the using the following? Avoids the need for /# and tabindex, and works around the unwanted spaces.

```diff
diff --git a/ts/import-log/QueueSummary.svelte b/ts/import-log/QueueSummary.svelte
index d054e66d9..01e1d2d9d 100644
--- a/ts/import-log/QueueSummary.svelte
+++ b/ts/import-log/QueueSummary.svelte
@@ -26,9 +26,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         </IconConstrain>
         {summary.summary_template({ count: notes.length })}
         {#if summary.canBrowse}
-            (
-            <a href="/#" tabindex="-1" on:click={onShow}>{tr.importingShow()}</a>
-            )
+            <button on:click={onShow}>{tr.importingShow()}</button>
         {/if}
     </li>
 {/if}
```